fhd_output/fhd_quickview.pro

```diff
@@ -28,6 +28,7 @@ IF N_Elements(beam_threshold) EQ 0 THEN beam_threshold=0.05
 IF N_Elements(beam_output_threshold) EQ 0 THEN beam_output_threshold=beam_threshold/2.
 IF N_Elements(image_mask_horizon) EQ 0 THEN image_mask_horizon=1
 IF status_str.fhd EQ 0 THEN model_recalculate=0
+IF ~keyword_set(image_filter_fn) THEN image_filter_fn='filter_uv_uniform'
```

Comment:
Minor change: please also print a warning message if `image_filter_fn` was not defined and it uses the default.